templates/blog.jade

```diff
@@ -5,40 +5,51 @@ block append title
 
 block main
   div(id="Blog")
+    
+    - var articles = env.helpers.getArticles(contents)
+    - var a0 = articles[0]
+    - var a0_img_src = (a0.metadata.featured_img ? a0.metadata.featured_img : "https://placehold.it/800x600")
+
+    div.Background-img-top--blog.Background-img-cover.u-padBottom--3
+      div(class="Background-img--full-wh" style="background-image: url('"+a0_img_src+"');")
+      div.Site-content-container.Background-img-caption-container.u-textCenter
+        a(href=a0.url)
+          div.Background-header-text= a0.title
+          p.Background-img-caption= moment.utc(a0.date).format('YYYY MMM DD')
+        if a0.metadata.subtitle
```

Comment:
@willpatera - On line 49-51, statements after `if` was not indented so i thought it didn't have to.Currently It doesn't seem to have any problems though but i guess indentation but be better.